packages/patternfly-4/react-table/src/components/Table/ActionsColumn.js

```diff
@@ -38,22 +40,22 @@ class ActionsColumn extends React.Component {
   };
 
   onSelect = (event, onClick) => {
-    const { rowId } = this.props;
+    const { rowData, extraParams } = this.props;
     event.preventDefault();
-    onClick && onClick(event, rowId);
+    onClick && onClick(event, rowData, extraParams);
```

Comment:
thanks for adding this!

Reply:
The rowId is already included in extraParams as rowIndex. Does it make a sense to include it separately? We are passing rowData in this fashion in other callbacks as well.

> So we don't break current API.

that is true, but this is already happening (e.g. onRowClick in https://github.com/patternfly/patternfly-react/pull/1227). I mean, aren't tables alpha at the moment?

Reply:
Not really they are already used by some projects and we shouldn't break the API for them.